packages/core/src/styles/style-controller.ts

```diff
@@ -68,6 +72,7 @@ export class StyleController {
   }
 
   public generateCssVariable(themeKey: string, propertyKey: string, value: string | number) {
+    if (value === undefined || value === 'undefined') {return '';}
```

Comment:
let's avoid same line return. It's bad for readability

Reply:
Good point. Will fix!

Reply:
@bigopon 
Fix: https://github.com/platform5/ux/commit/c65393cc5e4c3b649e345b6a4dc571e5fb7362e0